packages/garbo/src/tasks/post/index.ts

```diff
@@ -238,7 +238,7 @@ function fallbot(): GarboPostTask {
   };
 }
 
-function refillCinch(): GarboPostTask {
+export function refillCinch(): GarboPostTask {
```

Comment:
If we're doing this as part of post, as long as we're running post with our adventures, we don't need a separate task to do this.